### PR TITLE
Manage charges page react data table component

### PR DESCRIPTION
**Pull request by lionsdwarf**
billing_admin

### CODE REVIEW

**Comment by lionsdwarf** [comment]
Note that the gif uses stubbed query response data, but in reality there is a loading spinner while the charge rollups process....

**Comment by craigspaeth** [comment]
👏 👏 So exciting!

File: apps/manage/components/charge/index.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+          getCharges(d)
+        }}
+      />
+      <div>
+        {isFetching ? (
+          <div className={cx('spinner')} />
+        ) : charges.length ? (
+          <DataTable
+            title={`Charges for ${selectedDate.format('ddd, MMM D YYYY')}`}
+            expandableRows
```
Comment (craigspaeth) [inline apps/manage/components/charge/index.js]: Nice—cool to see how well this component worked for this use case.

File: apps/manage/lib/view-model/charge/domain.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
+import _ from 'lodash'
+import moment from 'moment'
+
+export const selectedCharges = charges => charges.filter(({ isSelected }) => isSelected)
+
+export const couponsTotal = coupons => !coupons || !coupons.length ?
```
Comment (craigspaeth) [inline apps/manage/lib/view-model/charge/domain.js]: 👍 well done moving some of this business logic to the view model. We'll want to be extra careful about test coverage on this view model.

**Comment by craigspaeth** [comment]
This is looking great! The CSS is a bit copy-pasta-messy but it gets the job done—I'll take a pass at cleaning that up.

Really impressed with the view model work here—very well broken up and architected :clap: Good stuff, excited to see the tests and such wrap up!

**Comment by lionsdwarf** [comment]
This has been updated. Please re-review. Thanks!

**Comment by craigspaeth** [comment]
Yay! So excited to merge this! 👏 🎉

**Pull request merged**